code/game/objects/items/storage/pouch.dm

```diff
@@ -123,7 +123,8 @@
 	desc = "It can contain a pistol. Useful for emergencies."
 	icon_state = "pistol"
 	max_w_class = 3
-	can_hold = list("/obj/item/weapon/gun/pistol")
+	can_hold = list("/obj/item/weapon/gun/pistol",
+					"/obj/item/weapon/gun/revolver")
```

Comment:
why does this need to be a another line

Reply:
Because basic readability? It's not my fault github has tabs = 8 spaces

Reply:
Multiline lists are fine